act-rules-format.bs

```diff
@@ -237,9 +231,10 @@ Outcome {#output-outcome}
 
 The procedure of a rule MUST always return one of the following outcomes:
 
-- **Passed**: The Test Target passed the test
-- **Failed**: The Test Target failed the test
-- **CannotTell**: The result of the test is unknown. Human intervention or further testing is needed.
+- **Passed**: All *expectations* for the Test Target were true
+- **Failed**: One or more *expectations* for the Test Target was false
+- **CannotTell**: The *applicability* or some of the *expectations* were inconclusive. Human intervention or further testing is needed.
```

Comment:
"Human intervention or further testing is needed" - This implies that the default mode for all tests are automated testing.

Reply:
I'd argue that we should remove `CannotTell` entirely.

Reply:
Perhaps **CannotTell** could be thought of as an interim response from automated testing. It is needed for that purpose, even though it may not be a final outcome like the other three.
My 2p.

Reply:
Dropping this particular result type for now.